api/capi/src/nnstreamer-capi-single.c

```diff
@@ -748,32 +758,49 @@ ml_single_invoke (ml_single_h single,
   in_data = (ml_tensors_data_s *) input;
   *output = NULL;
 
-  if (!single_h->filter || single_h->state >= JOIN_REQUESTED) {
+  if (!single_h->filter) {
     ml_loge ("The given param is invalid, model is missing.");
     status = ML_ERROR_INVALID_PARAMETER;
     goto exit;
   }
 
+  if (single_h->state >= JOIN_REQUESTED) {
+    ml_loge ("The thread is not idle.");
```

Comment:
Incorrect message. This means that the handle is closed or being closed. (refer to nnstreamer-capi-single.c:88)